actionpack/lib/action_controller/log_subscriber.rb

```diff
@@ -67,6 +67,16 @@ def #{method}(event)
       METHOD
     end
 
+    %w(write_fragment read_fragment).each do |method|
+      class_eval <<-METHOD, __FILE__, __LINE__ + 1
+        def #{method}(event)
+          return unless logger.info? && ActionController::Base.enable_fragment_cache_logging
```

Comment:
What do you think about using the config for every one of these fragment cache log methods?